### ISSUE TITLE
Design kit: update all status icons

### ISSUE THREAD

**Issue by oliviaflory** [carbon-design-system/carbon-for-ibm-design-kit#9]
Update all status icons to use `white` or `black` from the IDL color pallet on the inner fill for better accessibility.

![Screen Shot 2019-10-21 at 10.30.42 AM.png](https://images.zenhubusercontent.com/5cacfd2d41f23a1e95e068dc/e8e29618-8e22-4aec-ad11-3cde73496fba)

Additionally, the status icon color seems to be linked to multiple kits in some instances.

Acceptance criteria
- [x] all status icons are updated to white or back fill
- [x] all icons are using the IDL color pallet
- [x] white theme
- [ ] ~gray 10~
- [x] gray 90
- [ ] ~gray 100~
